doc/configuration.rst

```diff
@@ -1943,6 +1944,39 @@ Arguments:
   - board_config (str): optional, board config in the ``openocd/scripts/board/`` directory
   - load_commands (list of str): optional, load commands to use instead of ``init``, ``bootstrap {filename}``, ``shutdown``
 
+OpenFPGALoaderDriver
+~~~~~~~~~~~~~
+
+An :any:`OpenFPGALoaderDriver` controls the openFPGALoader tool to bootstrap an FPGA chip with a
+bitstream.
+
+A list of supported FPGA chips, boards and programming cables can be found on the project's page:
+- https://trabucayre.github.io/openFPGALoader/compatibility/fpga.html
+- https://trabucayre.github.io/openFPGALoader/compatibility/board.html
+- https://trabucayre.github.io/openFPGALoader/compatibility/cable.html
+
+Binds to:
+  interface:
+    - `AlteraUSBBlaster`_
+    - NetworkAlteraUSBBlaster
+    - `USBDebugger`_
+    - NetworkUSBDebugger
+
+Implements:
+  - :any:`BootstrapProtocol`
+
+.. code-block:: yaml
+
+  OpenFPGALoaderDriver:
+    board: vcu118
+    image: 'bitstream.bit'
+    frequency: 20000000
+
+Arguments:
+- image (str): optional, the default bitstream image file if not specified when calling load()
```

Comment:
```suggestion
- image (str): optional, key in :ref:`images <labgrid-device-config-images>` containing the path
  of the bitstream, if not specified when calling `load()`
```

Note that this key is *not* a path, but the name of the key in the `images:` section.